actions/fetchFilteredMemebers.ts

```diff
@@ -46,9 +46,9 @@ export const fetchFilteredMembers = async (term: string) => {
         longDescription: dummyMember.longDescription,
         joinDate: dummyMember.joinDate,
         isAdmin: dummyMember.isAdmin,
-        linkedInUrl: linkedIn || '',
-        githubUrl: github || '',
-        twitterUrl: '',
+        linkedInUrl: linkedIn || '#/',
+        githubUrl: github || '#/',
+        twitterUrl: '#/',
         _id: _id || '',
```

Comment:
> Can you explain this change please

Of course. there was a WARING while the card were loading because the link href property was blank ('')

You can read more about it [Here](https://stackoverflow.com/questions/52801051/react-site-warning-the-href-attribute-requires-a-valid-address-provide-a-valid)